perma_web/perma/celery_tasks.py

```diff
@@ -1224,3 +1228,88 @@ def conditionally_queue_internet_archive_uploads_for_date_range(start_date_strin
 
     else:
         logger.info("Skipped the queuing of file upload tasks: max tasks already in progress.")
+
+
+# WACZ CONVERSION
+
+def format_size(bytes_val):
```

Comment:
There is in fact a nice Django utility that does exactly this! We use it in [the Django admin](https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/admin.py#L399C16-L399C30) in a [few places](https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/admin.py#L631).

```
from django.template.defaultfilters import filesizeformat
```

How about we reuse that here instead of rolling our own, for consistency?